lola.html

```diff
@@ -0,0 +1,875 @@
+      <pre>
+
+{
+  "@context": ["https://www.w3.org/ns/actvitystreams",
+               {"@language": "en"}],
+  "id": "https://newsite.example.org/amy/items/02751cab-7dd7-416b-905b-...",
+  "previously": ["https://rhiaro.co.uk/2016/05/minimal-activitypub"],
+  "attributedTo": "https://newsite.example.org/amy/"
+  "to": ["https://rhiaro.co.uk/followers"],
+  "type": "Article",
+  "published": "2014-12-12T12:12:12Z",
+  "content": "&lt;div&gt;Article is in here... &lt;/div&gt;",
+}
+      </pre>
+
+      <h4>Example Copied Object in Outbox</h4>
+
+      <p>If the destination server chooses to place Create activities in the new outbox for some or all content items:</p>
+
+      <pre>
+{
+  "@context": ["https://www.w3.org/ns/actvitystreams",
+  "actor": "https://newsite.example.org/amy",
+  "id": "https://newsite.example.org/amy/items/e2495916-877c-4428-8ada-c199508ad855",
+  "to": [...],
+  "type": ["Copy", "Create"]
+
+  {
+    "@context": ["https://www.w3.org/ns/actvitystreams",
+                 {"@language": "en"}],
+    "id": "https://newsite.example.org/amy/items/02751cab-7dd7-416b-905b-...",
+    "previously": ["https://rhiaro.co.uk/2016/05/minimal-activitypub"],
+    "attributedTo": "https://newsite.example.org/amy/"
+    "to": ["https://rhiaro.co.uk/followers"],
+    "type": "Article",
+    "published": "2014-12-12T12:12:12Z",
+    "content": "&lt;div&gt;Article is in here... &lt;/div&gt;",
+  }
+}
+      </pre>
+
+      <p>A destination server might also reference the copied article rather than wrap it:</p>
+
+      <pre>
+{
+  "@context": ["https://www.w3.org/ns/actvitystreams",
+  "actor": "https://newsite.example.org/amy",
+  "id": "https://newsite.example.org/amy/items/e2495916-877c-4428-8ada-c199508ad855",
+  "to": [...],
+  "type": ["Copy", "Create"]
+  "object": "https://newsite.example.org/amy/items/02751cab-7dd7-416b-905b-...",
+  }
+}
+      </pre>
+
+      <h4>Example Copied Like</h4>
+
+      <p>
+        A like originating from a moved account may look like this when fetched - note the actor and ID are both dereferenceable for the new server, and the ‘previously’ value shows older IDs.
+      </p>
+      <pre>
+
+{
+  "@context": ["https://www.w3.org/ns/activitystreams",
+               {"@language": "en"}],
+  "type": "Like",
+  "actor": "https://newserver.example.org/chris/",
+  "id": "https://newserver.example.org/chris/like/1972"
+  "previously": ["https://dustycloud.org/chris/like/228"]
+  "name": "Chris liked 'Minimal ActivityPub update client'",
+  "object": "https://rhiaro.co.uk/2016/05/minimal-activitypub",
+  "published": "2014-12-14T14:14:14Z",
+  "to": ["https://rhiaro.co.uk/#amy",
+         "https://dustycloud.org/followers",
+         "https://rhiaro.co.uk/followers/"],
+  "cc": "https://e14n.com/evan"
+}
+
+      </pre>
+
+      <p>
+        It is NOT strictly required of servers to add or maintain “previously” lists.  A user may wish to move content without associating with an old address for many reasons. 
```

Comment:
We'll need to harmonize "previously" with "alsoKnownAs", but I think that's a problem for another day.

Reply:
By harmonize do you mean rename previously to "alsoKnownAs" or keep them separate but consistent?  "alsoKnownAs" is more of a concurrent alias than previously, and there was some debate about whether implementations accept list values (https://socialhub.activitypub.rocks/t/defining-alsoknownas/907/16 is where i tried to catch up on this).  

I've also considered not only proposing 'previously' as a list, but also including other data.  Previously could be a list of previous URL, Actor and date copied values.  But I'm fighting against the instinct to be complete, and presently listening to the instinct to be simple.